lib/jsdom/living/nodes/HTMLTemplateElement-impl.js

```diff
@@ -2,13 +2,54 @@
 
 const HTMLElementImpl = require("./HTMLElement-impl").implementation;
 
+const Document = require("../generated/Document");
+const DocumentFragment = require("../generated/DocumentFragment");
+
 const { cloningSteps, domSymbolTree } = require("../helpers/internal-constants");
 const { clone } = require("../node");
 
+/* Symbol assigned to a document to mark it as inert */
+const INERT_TEMPLATE_DOCUMENT = Symbol("inhert-template-document");
+
+/* Symbol assigned to a document to store the associated inert template document. */
+const ASSOCIATED_INERT_TEMPLATE_DOCUMENT = Symbol("associated-inert-template-document");
```

Comment:
Why use symbols? I'd rather just do something like `doc._isInertTemplateDocument` and `doc._associatedInertTemplateDocument`.

Also don't use multiline comments for singleline comments.

Reply:
Since we are attaching properties on the document that are not defined in the constructor, my original thought was to create `Symbol` to prevent property clashing. Will make the change.